test/e2e/karmadactl_test.go

```diff
@@ -522,3 +522,79 @@ var _ = ginkgo.Describe("Karmadactl exec testing", func() {
 		}
 	})
 })
+
+var _ = ginkgo.Describe("Karmadactl top testing", ginkgo.Labels{NeedCreateCluster}, func() {
+	ginkgo.Context("Karmadactl top pod which does not exist", func() {
+		ginkgo.It("Karmadactl top pod which does not exist", func() {
+			podName := podNamePrefix + rand.String(RandomStrLength)
+			for _, clusterName := range framework.ClusterNames() {
+				cmd := framework.NewKarmadactlCommand(kubeconfig, karmadaContext, karmadactlPath, "", karmadactlTimeout, "top", "pod", podName, "-n", testNamespace, "-C", clusterName)
+				_, err := cmd.ExecOrDie()
+				gomega.Expect(err).Should(gomega.HaveOccurred())
```

Comment:
Can the returned error message provide more detailed information for judgment, such as including the field "does not exist"? This way, the judgment may be more precise. Otherwise, it may encounter other errors.

Reply:
done~
```go
			for _, clusterName := range framework.ClusterNames() {
				cmd := framework.NewKarmadactlCommand(kubeconfig, karmadaContext, karmadactlPath, "", karmadactlTimeout, "top", "pod", podName, "-n", testNamespace, "-C", clusterName)
				_, err := cmd.ExecOrDie()
				gomega.Expect(strings.Contains(err.Error(), fmt.Sprintf("pods \"%s\" not found", podName))).To(gomega.BeTrue(), "should not found")
			}
```